### PR TITLE
feat(agg): add link and descriptions for the $densify stage COMPASS-5177

### PR DESCRIPTION
**Pull request by alenakhineika**
## Description

Add link and descriptions for the `$densify` stage.

### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [x] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [x] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/compass-aggregations/src/constants.js [hunk 1]
```diff
@@ -92,14 +92,18 @@ export const STAGE_SPRINKLE_MAPPINGS = {
...
+    link: 'https://docs.mongodb.com/rapid/reference/operator/aggregation/densify',
+    tooltip: 'Creates new documents in a sequence of documents where certain values in a field are missing.'
   },
   $documents: {
     link: 'https://docs.mongodb.com/v5.1/reference/operator/aggregation/documents/#mongodb-pipeline-pipe.-documents',
     tooltip: 'Returns literal documents from input values.'
   },
+  $facet: {
+    link: 'https://docs.mongodb.com/manual/reference/operator/aggregation/facet/#pipe._S_facet',
+    tooltip: 'Allows for multiple parellel aggregations to be specified.'
```
Comment (addaleax) [inline packages/compass-aggregations/src/constants.js:105]:
```suggestion
    tooltip: 'Allows for multiple parallel aggregations to be specified.'
```

File: packages/compass-aggregations/src/constants.js [hunk 2]
```diff
@@ -92,14 +92,18 @@ export const STAGE_SPRINKLE_MAPPINGS = {
...
+  $densify: {
+    link: 'https://docs.mongodb.com/rapid/reference/operator/aggregation/densify',
+    tooltip: 'Creates new documents in a sequence of documents where certain values in a field are missing.'
   },
   $documents: {
     link: 'https://docs.mongodb.com/v5.1/reference/operator/aggregation/documents/#mongodb-pipeline-pipe.-documents',
     tooltip: 'Returns literal documents from input values.'
   },
+  $facet: {
+    link: 'https://docs.mongodb.com/manual/reference/operator/aggregation/facet/#pipe._S_facet',
```
Comment (addaleax) [inline packages/compass-aggregations/src/constants.js:104]:
```suggestion
    link: 'https://docs.mongodb.com/manual/reference/operator/aggregation/facet/',
```

(it doesn’t look like that anchor exists on the page?)

**Review by addaleax — APPROVED** [review]

File: packages/compass-aggregations/src/constants.js (hunk 2 above)
Comment (alenakhineika) [reply packages/compass-aggregations/src/constants.js:104]: I haven't added facet in this pr, just moved it in code. And also I can see that almost all stages has this postfix. Do you know why we use it or why not should be using it?

Comment (addaleax) [reply packages/compass-aggregations/src/constants.js:104]: Alright, then let's leave it :) I don't know why we have those anchors, I assume at some point they had some relevance that has now disappeared.

**Pull request merged**